alephclient/crawldir.py

```diff
@@ -58,12 +70,15 @@ def is_excluded(self, path: PathLike) -> bool:
         # The exclude pattern is constructed bearing in mind that will
         # be called using fullmatch.
         if self.exclude is None:
-            return false
+            return False
+
+        path = Path(path)
+
         if path.is_dir():
-            return self.exclude["d"].fullmatch(path.name)
-        return self.exclude["f"].fullmatch(path.name)
+            return self.exclude["d"].match(path.name) is not None
+        return self.exclude["f"].match(path.name) is not None
```

Comment:
`match` is not going to behave correctly here. eg: it will exclude something like `desktop.ini.pdf` or `thumbs.db3` - which is not the expected behaviour. We want `fullmatch` here.